RevenueCatUI/Templates/V2/Components/Stack/StackComponentView.swift

```diff
@@ -462,22 +477,22 @@ struct StackComponentView_Previews: PreviewProvider {
                         components: [
                             .stack(.init(
                                 components: [],
-                                size: .init(width: .fixed(300), height: .fixed(50)),
+                                size: .init(width: .fixed(50), height: .fixed(50)),
```

Comment:
Had to tweak the fixed height of some previews and increase them by the amount of padding in that axis so snapshots would not change.